src/client/app/needs/new_need_step_1.html

```diff
@@ -1,137 +1,27 @@
-                                <ul class="list-unstyled" role="tablist">
-                                    <li>
-                                        <a href="#">Дитячий одяг</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Жіночий одяг</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Чоловічий одяг</a>
-                                    </li>
-                                </ul>
-                            </div>
-                            <div class="hidden subcategory_child">
-                                <!-- Sub-categories, hidden, should be displayed after main category is selected -->
-                                <ul class="list-unstyled" role="tablist">
-                                    <li>
-                                        <a href="#">Верхній одяг</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Светри</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Штани</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Футболки</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Білизна</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Для немовлят</a>
-                                    </li>
-                                    <li>
-                                        <a href="#">Інше</a>
-                                    </li>
-                                </ul>
-                            </div>
-                        </div>  
-                        <div class="modal-footer">
-                            <button type="button" class="btn btn-warning" data-dismiss="modal">Назад</button>
-                            <button type="button" class="btn btn-primary">Підтвердити</button>
-                        </div>
-                    </div>
-                </div>
-            </div>
-            <div class="btn-group row pull-right">
-                <div class="col-sm-12">
-                    <button type="button" class="btn btn-default">Назад</button>
-                    <button type="submit" class="btn btn-primary col-sm-offset-1">Далі</button>
-                </div>
-            </div>
-        </form>
-    </div>
+  <div class="container">
+      <h2 class="text-left text-heading">Нова потреба</h2>
+      <form class="form-horizontal" name="NewNeed" ng-submit="vm.submitNeed(vm.need.title, vm.need.category)">
```

Comment:
why pass these as parameters? don't overload the DOM with extra info. just access these inside of the method.